cogs/induct.py

```diff
@@ -181,6 +181,13 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
             name="introductions",
         )
 
+        initial_response: discord.Interaction | discord.WebhookMessage = await ctx.respond(
+            (
```

Comment:
String does not need to be spread by brackets when it does not fall onto multiple lines.